cloud/rackspace/rax_images.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# This is a DOCUMENTATION stub specific to this module, it extends
+# a documentation fragment located in ansible.utils.module_docs_fragments
+DOCUMENTATION = '''
+---
+module: rax_images
+short_description: Manipulate Rackspace Cloud Server Images
+description:
+  - Manipulate Rackspace Cloud Server Images
+version_added: "2.0"
+options:
+  name:
```

Comment:
Please collapse `id` and `name` into a single `server` such as what is done in `rax_cbs_attachments`:  https://github.com/ansible/ansible-modules-core/blob/devel/cloud/rackspace/rax_cbs_attachments.py#L37-L41